mobify/platform-scaffold
# set loggedin to false when user is signed out

sign out doesn't get updated until you browse to another page

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1391

## Changes
- set loggedin to false in signOut function

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Open menu and sign in
- Open menu and sign out, check the text is back to sign in
